README.md

```diff
@@ -69,15 +50,75 @@ NAME                                        READY   STATUS    RESTARTS   AGE
 metacontroller-0                            1/1     Running   0          2m31s
 ```
 
-Finally, test the operator by deploying the
-IntegrationRoute [example](operator%2Fexample%2FREADME.md).
+### Customizing Your keip Container
+Note: If this is your first time through this installation process, it may be helpful to return to this step later. This
+is generally needed for all but the simplest deployments, but it isn't necessary if you're just familiarizing yourself 
+with the installation process.
+
+The default keip container provides only the basic components for Spring Integration. To fully utilize the potential of 
+your integration routes, you will need to include additional Spring Integration components or your own Java code. See 
+[README.md](keip-container-archetype%2FREADME.md) for instructions on how to create a custom container.
+
+Once your new container is available, you'll need to set that name in the keip-controller-props ConfigMap. The name of 
+this ConfigMap includes a hash, so we'll need to retrieve the full ConfigMap name first. We'll need to change the value
+for the integration-image key.
+
+Note that the single-line command below requires that you have the `jq` utility installed.
+
+```shell
+kubectl edit cm `kubectl get configmap --all-namespaces -o json | jq -r '.items[] | select(.metadata.name | startswith("keip-controller-props")) | .metadata.name'`
```

Comment:
:question: If the webhook is already running, will editing the `keip-controller-props` configmap immediately take effect? Or do we need to restart the webhook pod?

Reply:
Added a step to restart the webhook.